Makefile

```diff
@@ -74,6 +74,10 @@ build: $(BIN)/$(EXECUTABLE)
 
 $(BIN)/$(EXECUTABLE): $(wildcard *.go)
 	go build -v -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
+	cp -r ./templates ./bin/
+	cp -r ./public ./bin/
+	cp -r ./conf ./bin/
+	cp -r ./scripts ./bin/
 
```

Comment:
@tboerger or build on bin and make a symbolic link on root folder. I will test here to see if works.